src/browser/index.js

```diff
@@ -1,3 +1,3 @@
 export { assert, aTimeout, defineCE, expect, html, nextFrame, oneEvent, waitUntil } from '@open-wc/testing';
-export { focusWithKeyboard, focusWithMouse } from './focus.js';
+export { clickAt, clickElem, focusElem, hoverAt, hoverElem, sendKeys, sendKeysElem } from './commands.js';
```

Comment:
I wonder if we should add visual-diff tests for these in here eventually, or if that's overkill because they'll be used in tests all over the consumer repos anyways 🤔

Reply:
Good call -- I've added unit tests (not visual diff) for the commands, since I'd rather catch bugs in here vs. in consumer code. They're not _super_ useful since they're mostly testing the `test-runner-commands` stuff, but meh.